doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -758,42 +801,205 @@ Response:
             "value": 21.7
         }
 
-### Get a single attribute value. Not exists [GET /v2/entities/{entityId}/attrs/{attrId}]
+### Update attribute data [PUT /v2/entities/type/{entityType}/id/{entityId}/attrs/{attrName}]
```

Comment:
mmmm I'm not sure we need to have PUT verb here, but that is part of a separate discussion on getting rid of /type/t/id/i